src/core/getters/scalar.mock.ts

```diff
@@ -81,7 +81,7 @@ export const getMockScalar = async ({
     case 'number':
     case 'integer': {
       return {
-        value: getNullable('faker.datatype.number()', item.nullable),
+        value: getNullable(`faker.datatype.number({min: ${item.minimum}, max: ${item.maximum}})`, item.nullable),
```

Comment:
Thanks for the pull request. You also need to handle the case when it's not defined in the specification

Reply:
Of course! It's my first attempt at contributing to anything open source so i'm excited. For the case where it is not defined in the specification, I mentioned in the top comment it will simply pass `undefined` to the faker function which will lead to the same default behavior that was already happening (it will use the default values for min and max which are 0 and 99999). Is this sufficient or should I handle it some other way?